### PR TITLE
Introduce basic docker images

### PR DESCRIPTION
**Pull request by mholthausen**
This PR

- adjusts the docker compose file to use the images provided via ghcr.io
- introduces basic Docker images for n8n and rabbitMQ
- **Note**: the postgres files from an intermediate state for initialization have been removed. Please continue to use the files from the `workflows` folder and import them into n8n

@weskamm please review

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -10,8 +10,14 @@ services:
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
       - POSTGRES_DBNAME=n8n
     volumes:
-      - ./postgresql_init_data:/docker-entrypoint-initdb.d
-      - ./postgresql_data:/var/lib/postgresql/data:Z
+      - ./n8n-postgres/postgresql_init_data:/docker-entrypoint-initdb.d
+      - ./n8n-postgres/postgresql_data:/var/lib/postgresql/data:Z
+  rabbitmq:
```
Comment (weskamm) [inline docker/docker-compose.yml:15]: This repo was meant to hold only the n8n relevant things (so please remove rabbitmq and worker containers). 
Please create a new repository which only holds a docker-compose file which combines all the different components:
  * rabbitmq
  * n8n
  * needed databases
  * worker images

That way we can keep things separated

**Comment by weskamm** [comment]
Sorry for confusion, you were missing some background information, see comment above. Otherwise changes looking good!

File: docker/docker-compose.yml (hunk 1 above)
Comment (mholthausen) [reply docker/docker-compose.yml:15]: Moved the components to [klips-project/rabbitmq-n8n](https://github.com/klips-project/rabbitmq-n8n).

**Comment by mholthausen** [comment]
This PR is closed in consideration to #5 . The comments noted above have been implemented.

**Pull request closed without merging**